packages/neuron-wallet/src/store/WalletStore.ts

```diff
@@ -1,85 +1,95 @@
-import BaseStore from './store'
+import Store from 'electron-store'
+import { v4 } from 'uuid'
 import { Keystore } from '../keys/keystore'
 import env from '../env'
 
-const keyWalletName = 'WalletName'
+const encryptKey = 'Neuron'
+
+const WalletIDKey = 'WalletID'
```

Comment:
better to be in env

Reply:
It is a key just used in wallet store.So I don't think it needs to go to env

Reply:
Let's do it this way: check env, and use different root folder for keys.

For example, for production, use `/keys` (relative to default user data folder), for development, use `/keys-dev`, for test, use `/keys-test`. This is to avoid wiping local data when running tests or affecting production data when running in development on the same machine.

Reply:
ok,test is running in different folder now.I will move dev part to new folder